v0.0.20251104

mjuric opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

mjuric committed:
Version 0.0.20251104

mjuric committed:
Merge branch 'conda-forge:main' into main

mjuric commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I failed to even lint the recipe, probably because of a conda-smithy bug :cry:. This likely indicates a problem in your `meta.yaml`, though. To get a traceback to help figure out what's going on, install conda-smithy and run `conda smithy recipe-lint --conda-forge .` from the recipe directory. You can also examine the [workflow logs](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19106621263) for more detail.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19106621263](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19106621263). Examine the logs at this URL for more detail.</sub>

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.11.05.12.33.0

mjuric committed:
linter fix

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

mjuric commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.11.05.12.33.0

mjuric committed:
Fix build

mjuric committed:
Fix build

mjuric committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.11.05.12.33.05

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

mjuric committed:
Further fixes

mjuric commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19110059717](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19110059717). Examine the logs at this URL for more detail.</sub>

mjuric committed:
More

mjuric committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.11.05.12.33.05

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

mjuric committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.11.05.12.33.05

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

mjuric committed:
Platform updates

mjuric committed:
Foo

mjuric committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.11.05.12.33.05

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

mjuric committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.11.05.12.33.05

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

mjuric committed:
Foo

mjuric committed:
More fixes

mjuric committed:
FIX

mjuric committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.11.05.12.33.05
